Dangerfile

```diff
@@ -0,0 +1,44 @@
+# Get PR number and initialize is_good_pr variable
+pr_number = github.pr_json["number"]
+is_good_pr = true
+
+# Report if number of changed lines is > 500
+if git.lines_of_code > 500
+  warn("Number of updated lines of code is too large to be in one PR. Perhaps it should be separated into two or more?")
+  auto_label.set(pr_number, "Big PR", "ffff00")
```

Comment:
These colours are a bit eye-catching for labels. Here's some colour values from the github default colour set, which will harmonise better with our existing labels:

#B60205 - red
#D93F0B - orange
#FBCA04 - yellow

However, I've also checked the source code for the danger-auto_label plugin, and the colour values are ignored if the label already exists. This means that we can change them in future without them being overridden by this code, so maybe it doesn't matter what we set them to here.

Reply:
Markdown'ified:
`#B60205` - red
`#D93F0B` - orange
`#FBCA04` - yellow